helper/plan.php

```diff
@@ -705,33 +755,39 @@ protected function findAffectedPages($src, $class, $type) {
     /**
      * Find missing pages in the $src namespace
      *
-     * @param $src
-     * @param $dst
+     * @param string $src source namespace
+     * @param string $dst destination namespace
+     * @param int    $type either self::TYPE_PAGES or self::TYPE_MEDIA
      */
-    protected function findMissingPages($src, $dst) {
-        if(is_null($this->referenceidx)) {
-            global $conf;
-            // FIXME this duplicates Doku_Indexer::getIndex()
-            $fn = $conf['indexdir'].'/relation_references_w.idx';
-            if (!@file_exists($fn)){
-                $this->referenceidx = array();
-            } else {
-                $this->referenceidx = file($fn, FILE_IGNORE_NEW_LINES);
-            }
+    protected function findMissingDocuments($src, $dst, $type = self::TYPE_PAGES) { //FIXME: expects $src without colon, but $dst with colon
```

Comment:
re the FIXME. any way to either check for correct input or fix it automatically?